core/src/commonMain/kotlin/entity/User.kt

```diff
@@ -37,6 +37,13 @@ public open class User(
     override val id: Snowflake
         get() = data.id
 
+    /**
+     * Returns true if the user is the same as the bot.
+     */
+    public val isSelf: Boolean by lazy {
+        id == kord.selfId
+    }
+
```

Comment:
```suggestion
public val isSelf: Boolean get() = id == kord.selfId
```

I think the lazy actually causes worse performance in this case because it needs to allocate a bunch of objects

Reply:
Changed it - makes sense